node/node.go

```diff
@@ -179,16 +180,21 @@ func GetNodeServiceCommand() *cobra.Command {
 				return fmt.Errorf("ensure folders exist: %w", err)
 			}
 
-			if err := app.Lock(); err != nil {
+			unlock, err := lock(conf.FileLock)
+			if err != nil {
 				return fmt.Errorf("getting exclusive file lock: %w", err)
 			}
-			defer app.Unlock()
+			defer func() {
+				if err := unlock(); err != nil {
+					lg.Error("failed to unlock file", zap.String("path", conf.FileLock), zap.Error(err))
```

Comment:
The default logger is a sugared logger and will print weird stuff when used with `zap.Field`s

```suggestion
					lg.Zap().Error("failed to unlock file", zap.String("path", conf.FileLock), zap.Error(err))
```